packages/cdk-ecs-codedeploy-resources/lib/index.ts

```diff
@@ -0,0 +1,268 @@
+import * as cdk from "@aws-cdk/core";
+import * as customresource from "@aws-cdk/custom-resources";
+import * as iam from "@aws-cdk/aws-iam";
+import * as s3 from "@aws-cdk/aws-s3";
+import * as cloudfront from "@aws-cdk/aws-cloudfront";
+
+export interface CdkEcsCodedeployResourcesProps {
+  stage: string;
+  clusterName: string;
+  serviceName: string;
+  appName: string;
+  listenerArn: string;
+  targetGroupName: string;
+  commitHash: string;
+}
+
+export class CdkEcsCodedeployResources extends cdk.Construct {
+
+  constructor(scope: cdk.Construct, id: string, props: CdkEcsCodedeployResourcesProps) {
+    super(scope, id);
+
+    const iamRole = iam.Role.fromRoleArn(this, "EcsBlueGreenDeploymentResourcesRole", "arn:aws:iam::022393549274:role/AWSCustomResourceRoleFullAccess")
```

Comment:
Is that role available in the account already or did you create it? If it's available to all accounts you should string format it by looking up the account id. Regardless we don't want hardcoded ARNs in this.

Reply:
I created this role so it wouldn't be available to all accounts.  That being said what do we want to do with this role written in code going forward?  Are you just saying to make the account id in the arn dynamic?